examples/fuse_examples/imaging/classification/mnist/runner.py

```diff
@@ -258,10 +258,6 @@ def run_infer(paths: dict, infer_common_params: dict):
     lgr.info(f'infer_filename={os.path.join(paths["inference_dir"], infer_common_params["infer_filename"])}', {'color': 'magenta'})
 
     ## Data
-    transform = transforms.Compose([
```

Comment:
it's not needed anymore for the mnist example?

Reply:
It is in fuseimg/datasets/mnist.py.
This is a leftover I forgot to delete.